builds/e2e/templates/e2e-setup-base-image-update-release.yaml

```diff
@@ -0,0 +1,160 @@
+
```

Comment:
What are the main differences b/w this yaml and e2e-setup.yaml? It may be worthwhile to state these differences in comments here. This would be especially useful if we plan to consolidate them in a future effort.

Reply:
1. The image tagging is different
2. e2e-setup.yaml relies on artifacts & builds packages as a part of the pipeline, whereas the other doesn't.
3. e2e-setup.yaml defaults to Test/Debug/EdgeBuilds whereas the other is the exact opposite in all areas.

I'd argue that these are 2 separate files, despite being 60% same and that it's likely better that we _don't_ consolidate at least this particular pair of files.